source/mongoexport.txt

```diff
@@ -16,6 +16,9 @@
    :depth: 1
    :class: singlecol
 
+.. meta:: 
+  :description: mongoexport is a database tool that produces a JSON or CSV export of data stored in a MongoDB instance.
```

Comment:
For parity with the other descriptions:

```suggestion
  :description: mongoexport is a MongoDB database tool that produces a JSON or CSV export of data stored in a MongoDB instance.
```

Reply:
@sarahemlin Would it matter if the word MongoDB is used twice in a meta description?

Reply:
@ltran-mdb2 I think it's best to avoid redundancy and make these succinct.

Reply:
thanks @sarahemlin ! For this reason I'm going to leave the sentence as is